hivemind_redis_database/__init__.py

```
@@ -96,9 +436,73 @@ def __iter__(self) -> Iterable['Client']:
             An iterator over the clients in the database.
         """
         for client_id in self.redis.scan_iter(f"client:*"):
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)